services/api/src/models/definitions/user.json

```diff
@@ -46,7 +47,8 @@
     },
     "isDeveloper": {
       "type": "Boolean",
-      "default": false
+      "default": false,
+      "writeScopes": "users"
```

Comment:
whats the right value here ? to only allow admins to update this field ? 
I am thinking i not really sure how that works

Reply:
I am just gonna revert this, so this feature is more atomic.